sdk/eventhub/Azure.Messaging.EventHubs/src/Processor/Checkpoint.cs

```diff
@@ -12,6 +12,13 @@ namespace Azure.Messaging.EventHubs.Processor
     ///
     public class Checkpoint
     {
+        /// <summary>
+        ///   The fully qualified host name for the Event Hubs namespace this checkpoint is
+        ///   associated with.
+        /// </summary>
+        ///
+        public string FullyQualifiedNamespace { get; }
```

Comment:
I'll echo the request from Shivangi here and elsewhere.   I'd love to see the same example that we used in the `EventHubClient` to help raise awareness of what this value should look like.     I know this property is echoing back what was received, but I think there's value in having that example blurb associated with parameters and members so that its very "in your face" since it's not an obvious concept otherwise.